portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase

```diff
@@ -404,6 +404,77 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-131529."
+	@priority = 4
+	test NoXSSVulnerabilityInMBAttachment {
+		property test.assert.warning.exceptions = "false";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		var baseDir = PropsUtil.get("test.base.dir.name");
+		var dependenciesDir = PropsUtil.get("test.dependencies.dir.name");
+
+		var filePath = "${baseDir}${dependenciesDir}/test.html";
+		var content = '''<html>
+		<h1>Alert</h1>
+		<script>alert(1)</script>
+		</html>''';
+
+		FileUtil.write(${filePath}, ${content});
```

Comment:
Can we use [this file](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/dependencies/xss_test.html) in our test? 

If it works for the test, We just need to copy it into [the common dependencies folder](https://github.com/liferay/liferay-portal/tree/master/portal-web/test/functional/com/liferay/portalweb/dependencies).
If it doesn't, I think it's better to create a file and place it in the dependencies folder, so that it doesn't need to be created every time when running the test.